src/style.css

```diff
@@ -13,6 +13,15 @@ section {
 h1 {
   padding: 21px;
   margin-left: 35px;
+  background: -webkit-linear-gradient(red, rgb(74, 85, 86));
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  position: relative;
+  line-height: 0.9;
+  font-size: 55px;
+  font-weight: bold;
+
+  /* padding: 2rem; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.